src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -2,12 +2,8 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
 <%@ attribute name="buttonType" required="true" %>
-<a class="btn ${buttonType} btn-tm-actions session-${publishButton.actionNameLowercase}-for-test"
+<a class="session-${publishButton.actionNameLowercase}-for-test"
```

Comment:
The removal of `btn ${buttonType} btn-tm-actions` is causing issues on the publish/unpublish button on instructor feedback results page.
![image](https://cloud.githubusercontent.com/assets/6365864/26233560/d42a45ce-3c90-11e7-85ff-b27bfbf51e05.png)

Since the other tag file (sessionPanel.tag) using this passes the publishButton's own variable as the buttonType variable, you could do some refactoring to remove buttonType as an attribute here and possibly add a boolean attribute to indicate if this button is an inner button. Using that boolean attribute, you can determine whether `btn ${buttonType} btn-tm-actions` should be added into the class, and whether the tooltip placement should be top or left/right.